Packs/AWS-GuardDuty/Integrations/AWSGuardDutyEventCollector/AWSGuardDutyEventCollector.py

```diff
@@ -236,14 +235,13 @@ def main():  # pragma: no cover
                 })
 
             if argToBoolean(demisto.args().get('should_push_events', 'true')):
-                send_events_to_xsiam(events, params.get('vendor', 'AWS'), params.get('product', 'GuardDuty'))
+                send_events_to_xsiam(events, 'aws', 'guard_duty')
 
         elif command != 'test-module':
             raise NotImplementedError(f"Command {command} is not implemented.")
 
     except Exception as e:
-        return_error(f'Failed to execute {demisto.command()} command in AWSGuardDutyEventCollector.\nError:\n{str(e)}'
-                     f'\nTraceback:\n{traceback.format_exc()}')
+        return_error(f'Failed to execute {demisto.command()} command in AWSGuardDutyEventCollector.\nError:\n{str(e)}')
```

Comment:
I see you removed the traceback, do we see it in case of error?

Reply:
I noticed when running the test module, the printed traceback looks bad and it just makes it uncomfortable to see the error. When running the `Run advanced test and download a full report`, a traceback is printed automatically. The problem occurs when exceptions are being raised for other commands. No traceback is printed in the logs there. (Or at least I didn't find one)

I think for now, it's more important to have a clear error for the test module than a traceback for debugging. Although the fact that it's a tradeoff should raise a flag.